index.js

```diff
@@ -229,7 +229,7 @@ async function create(vm) {
     console.log(patchDocument);
   }
 
-  let authHandler = azdev.getPersonalAccessTokenHandler(vm.env.adoToken);
+  let authHandler = azdev.getHandlerFromToken(vm.env.adoToken);
```

Comment:
See https://github.com/microsoft/azure-devops-node-api/blob/master/api/WebApi.ts#L72 for more info

Reply:
@AaronCrawfis I assume you have tested this out and it all works?

Reply:
Yep, tested both managed identity:
![image](https://github.com/danhellem/github-actions-issue-to-work-item/assets/54363786/aa4e69d2-74f9-4e28-9033-d8593d4d08d5)

and PAT:
![image](https://github.com/danhellem/github-actions-issue-to-work-item/assets/54363786/25213f66-5cd9-4278-bf77-8223cfa48910)